monitoring/heartbeats/install-plugin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Install Heartbeats Plugin
+summary: Enabling endpoint instance monitoring by installing the Heartbeats plugin
+reviewed: 2018-01-05
+component: Heartbeats
+versions: 'Heartbeats:*'
+---
+
+To install the Heartbeats plugin into an endpoint add the following to the endpoint configuration:
+
+snippet: HeartbeatsNew_Enable
+
+WARN: When installing the heartbeat plugin, also configure each instance with a deterministic [Host Identifier](/nservicebus/hosting/override-hostid.md). This identifier is used to keep track of which instance is sending heartbeat messages to the ServiceControl instance.
```

Comment:
Why? The default is already endpoint name + version + path + host or something right?

Reply:
Which is fine in a desktop, on-premise environment. In a virtualized/cloud one it is rubbish. We have even seen cases where customers want to deploy each new version using Octopus Deploy and it creates new folders which messes up our default InstanceId algorithm.

I think we might need a separate page on instance identification within monitoring, then we can just point to that.

Reply:
To what does *[Host Identifier]* even belong? It seems to something stretching over multiple capabilities as a reference.

Reply:
I think it's a missing (or broken) concept in the domain: Endpoint instance identification. If we had that, then all of the capabilities would use it.